src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -202,6 +202,7 @@ namespace Akka.Actor
             public override int GetHashCode() { }
         }
     }
+    [System.ObsoleteAttribute("Actor producer pipeline API will be removed in v1.5.")]
```

Comment:
Why v1.5? We could remove it in 1.4 or in 1.6. Actually in some next major version

Reply:
I think it won't hurt if we give users more time for adoption. I hoped to bring native DI with v1.5, there is no immediate need to close this part of an API before that.